### PR TITLE
Hide imports, toc and many other fixes

### PR DESCRIPTION
**Pull request by jonaprieto**
This PR improves the website in the following way.
- Adds support for Katex by using `mdbook-katex`.
- Add missing files: Yes to the question stated here: 
   - #477.
- Hide imports by default
- Add table-of-contents on each page by using `mdbook-pagetoc`.
- Title in the menu bar is now a link to the home page
- CSS improvements
- Update the contributor's page.
- Update the CI to support the two first changes.
- Remove trailing whitespaces
- Remove double blank lines
- Fixes in SUMMARY.lagda.md (missing entries)

The resulting webpage would be like this:
https://jonaprieto.github.io/agda-unimath

PS. Sorry for the huge delta here; whoever reviews this. :P

### CODE REVIEW

File: src/CONVENTIONS.lagda.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 ```
 5. Immediately after the header, there should be a block of Agda code that loads the options, declares the present module, and performs all the imports. In particular, there should be no further imports later on in the file.
```
Comment (fredrik-bakke) [inline src/CONVENTIONS.lagda.md:11]: Mention the new import tags

File: src/CONTRIBUTORS.lagda.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ the library. In alphabetical order, by GitHub username:
 - [Matej Jazbec (MatejJazbec)](https://github.com/MatejJazbec)
 - [Alice Laroche (Seiryn21)](https://github.com/Seiryn21)
 - [Nathan van Doorn (Taneb)](https://github.com/Taneb)
+- [VictorBlanchi (VictorBlanchi)](https://github.com/VictorBlanchi)
```
Comment (fredrik-bakke) [inline src/CONTRIBUTORS.lagda.md:15]: Should this be `[Victor Blanchi (VictorBlanchi)]` ?

File: docs/Agda.css [hunk 3]
```diff
@@ -1,17 +1,17 @@
 /* Aspects. */
 .Agda .Comment {
-    color: #B22222
+    color: #948e8e
 }
 
 .Agda {
-    background: aliceblue;
     border: 1px solid #f0f0f0;
-    font-family: Menlo, Source Code Pro, Consolas, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace, serif;
```
Comment (fredrik-bakke) [inline docs/Agda.css:9]: Now the middle dot and the bullet characters look completely identical on my machine as well. I think it is still necessary to specify fonts (and perhaps distribute certain fonts. See discussion at #464)

Comment (fredrik-bakke) [reply docs/Agda.css:9]: The same goes for code blocks that have not been tagged as Agda code.

File: src/SUMMARY.lagda.md [hunk 4]
```diff
@@ -62,7 +62,6 @@
   - [Slice precategories](category-theory.slice-precategories.md)
```
Comment (fredrik-bakke) [inline src/SUMMARY.lagda.md:62]: At the top of this file (lines 5 and 6), there are two links to `README.md` that are broken to me. Apparently, these are not broken to Egbert. Do you know why this is/how to fix it?

**Review by fredrik-bakke — COMMENTED** [review]
This isn't a full review, but at least I had a look at the meta-files.

Also, I'd like to remark, although this is just a personal opinion, that I liked it when the imports were visible by default. This gave me easy access to links to all the relevant files to the current one. Especially the `foundation-core` files that are imported in their `foundation` counter-part.

**Comment by fredrik-bakke** [comment]
I hope this helps, and also, the web page is really shaping out! 😄

File: src/CONTRIBUTORS.lagda.md (hunk 2 above)
Comment (fredrik-bakke) [reply src/CONTRIBUTORS.lagda.md:15]: Oh, forget it. I see you're just using their GitHub display name

File: docs/Agda.css (hunk 3 above)
Comment (jonaprieto) [reply docs/Agda.css:9]: Let's fix this in another PR. I have unified the font for code and pre-blocks.

File: src/CONVENTIONS.lagda.md (hunk 1 above)
Comment (jonaprieto) [reply src/CONVENTIONS.lagda.md]: BTW, I reorganised the metafiles and put them inside the main folder. I think it's the best considering that they are not Agda files.

File: docs/Agda.css (hunk 3 above)
Comment (jonaprieto) [reply docs/Agda.css:9]: Certainly, issue #464 needs proper attention.

File: CONVENTIONS.md [hunk 5]
```diff
@@ -8,64 +8,21 @@
...
 `###` headings are for subsectioning the `## Definitions` and `## Properties` section. They are brief descriptions of the code block that follows. If the code block that follows is very long, we can even use `####` headings to subdivide the subsections.
 Note that `#` headings are only used for the title of the page.
-7. The subsections should have a markdown header of level `###` and they should concisely describe the content of the block of code that follows.
+1. The subsections should have a markdown header of level `###` and they should concisely describe the content of the block of code that follows.
```
Comment (fredrik-bakke) [inline CONVENTIONS.md:21]: Something strange has happened with the numbering here. Is this intentional?

File: src/SUMMARY.lagda.md (hunk 4 above)
Comment (jonaprieto) [reply src/SUMMARY.lagda.md:62]: Now, these two titles are links to `HOME.md`.

**Review by fredrik-bakke — APPROVED** [review]

File: CONVENTIONS.md [hunk 6]
```diff
@@ -76,3 +28,5 @@ At the end of the file you may also add a `See also` or `References` subsection
   This will be displayed as [Agda-Unimath](https://unimath.github.io/agda-unimath/).
 - If you want to reference another page without custom text but with a clickable link use the pattern ```<https://unimath.github.io/agda-unimath/>```.
   This will be displayed as <https://unimath.github.io/agda-unimath/>.
+
+One good example of the expected structure of a file is in the file [foundation.cantor-schroder-bernstein-escardo.lagda.md](https://raw.githubusercontent.com/UniMath/agda-unimath/master/src/foundation/cantor-schroder-bernstein-escardo.lagda.md).
```
Comment (fredrik-bakke) [inline CONVENTIONS.md:32]:
```md
[`foundation.cantor-schroder-bernstein-escardo.md`](foundation.cantor-schroder-bernstein-escardo.md)
```

File: CONVENTIONS.md [hunk 7]
```diff
@@ -1,73 +1,25 @@
...
-
-[ formalization of property 2]
-```
-
-At the end of the file you may also add a `See also` or `References` subsection where you reference related sources such as other modules or articles related to the contents of the file.
+Ideally, the first section of a file explains the idea, then proceeds to give the main definition that is the focus of the current file, then proceeds possibly with examples and by deriving basic properties of the defined concept. 
+8. At the end of the file you may also add a `See also` or `References` subsection where you reference related sources such as other modules or articles related to the contents of the file.
 
 - If you want to reference another module in the library use the pattern
   ```[`foundation.univalence-axiom`](foundation.univalence-axiom.md)```.
```
Comment (fredrik-bakke) [inline CONVENTIONS.md:25]: Also, I just realized this link is broken and should be
```md
[`foundation.univalence`](foundation.univalence.md)
```

File: CONVENTIONS.md (hunk 6 above)
Comment (fredrik-bakke) [reply CONVENTIONS.md:32]: Nevermind! What's the reasoning for using the raw file?

Comment (jonaprieto) [reply CONVENTIONS.md:32]: This is fine. I want a link to the RAW file so people can see the actual Markdown file.

**Comment by fredrik-bakke** [comment]
This PR has my approval now at least. Amazing work! 😁

**Pull request merged**